app/assets/stylesheets/views/_taxons.scss

```diff
@@ -54,11 +54,15 @@
 
       li {
         @include grid-column(1 / 3);
-        padding-bottom: $gutter;
+        padding-bottom: $gutter * 1.5;
```

Comment:
I think if you kept this as `$gutter` and added `padding-bottom: $gutter * 1.5;` into the `tablet` block you wouldn't need the `mobile` block.